packages/blockchain-api/test/blockscout.test.ts

```diff
@@ -70,11 +70,15 @@ describe('Blockscout', () => {
           "hash": "0xba620de2d812f299d987155eb5dca7abcfeaf154f5cfd99cb1773452a7df3d7a",
           "makerAmount": Object {
             "currencyCode": "cUSD",
+            "knownExchangeRates": undefined,
```

Comment:
It is confusing to me that the expected behavior here is to have an undefined exchange rate. Would it make sense to store the exchange rate for both transactions, even if it is redundant information? Or find some other way to avoid this undefined?